### PR TITLE
Enrich failed to parse line error message with feed_url and line

### PR DESCRIPTION
**Pull request by tomas321**
The `Failed to parse line.` error message is very frequent and missing more direct information about the specific line and feed.

### CODE REVIEW

File: intelmq/lib/bot.py [hunk 1]
```diff
@@ -813,7 +813,7 @@ def process(self):
                 else:
                     events = [value]
             except Exception:
-                self.logger.exception('Failed to parse line.')
+                self.logger.exception("Failed to parse line '{}' for feed '{}'".format(line, report['feed.url']))
```
Comment (wagner-certat) [inline intelmq/lib/bot.py]: You can't rely on the existence of `feed.url`.

Comment (tomas321) [reply intelmq/lib/bot.py]: Thanks for that. I fixed the issue in the new commit

**Comment by wagner-certat** [comment]
I'm not sure if I like that because then we have actual data (only for DEBUGging) in the ERROR log level.

File: intelmq/lib/bot.py [hunk 2]
```diff
@@ -813,7 +813,8 @@ def process(self):
                 else:
                     events = [value]
             except Exception:
-                self.logger.exception('Failed to parse line.')
+                self.logger.exception("Failed to parse line '{}' for feed '{}'".format(
+                                      line, report.get('feed.url', 'Feed URL is unspecified')))
```
[the remark this answers is not in the document]
Comment (tomas321) [reply intelmq/lib/bot.py:817]: Sorry for the longer wait on my reply. The more verbose log line was useful when we had many shadowserver feeds with repetitive errors of this type - it speeded up the diagnosis process. Although, I was not aware of the intelmqdump tool addressing this.

**Comment by wagner-certat** [comment]
Okay, makes sense. I'll close there then

**Pull request closed without merging**